.github/workflows/vm-lint.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     steps:
       - uses: actions/setup-node@v1
         with:
-          node-version: 8.x
+          node-version: 12.x
       - uses: actions/checkout@v1
       - name: Cache node modules
```

Comment:
There is a `key` value still based on Node 8 in this file.

Reply:
thanks will fix :)